interface/reports/collections_report.php

```diff
@@ -1033,7 +1054,7 @@ function checkAll(checked) {
 <?php
       if ($form_age_cols) {
         for ($c = 0; $c < $form_age_cols; ++$c) {
-          echo "  <td class='detail' align='right'>";
+          echo "<td class='detail ". $form_cb_with_debt && $balance<=0 ? "delete" : ""." align='right'>";
```

Comment:
Hi,
fixed it to : 

 echo "<td class='detail ". ($form_cb_with_debt && $balance<=0 ? "delete" : "")."' align='right'>";

thanks
Dror G.